examples/wheel/private/wheel_utils.bzl

```diff
@@ -0,0 +1,42 @@
+"""Helper rules for demonstrating `py_wheel` examples"""
+
+def _directory_writer_impl(ctx):
+    output = ctx.actions.declare_directory(ctx.attr.out)
+
+    args = ctx.actions.args()
+    args.add("--output", output.path)
+
+    for path, content in ctx.attr.files.items():
+        args.add("--file={}={}".format(
+            path,
+            json.encode(content),
+        ))
+
+    ctx.actions.run(
+        outputs = [output],
+        arguments = [args],
+        executable = ctx.executable._writer,
+    )
+
+    return [DefaultInfo(
+        files = depset([output]),
+        runfiles = ctx.runfiles(files = [output]),
+    )]
+
+directory_writer = rule(
+    implementation = _directory_writer_impl,
+    doc = "A rule for generating a directory with the requested content.",
```

Comment:
I think part of the point is also that a py_binary is used to do so, right? Because we're trying to show that, yes, the tool is actually run and works when run as a tool with RBE.

Let's put that in the doc here, too.

Reply:
I’m not aware of that behavior from py_binary. I’m happy to update the docs but I don’t think I have context on what you’re referring to. Could you add a suggestion snippet?

Reply:
Oh, I think I see now. This is just part of the `wheel_test` a bit further down? And it's just trying to cover that, if you feed it a generated directory, that things work. The particular program used to generate that directory and its content aren't important.

Ok, sorry; stet.

Reply:
😅 sorry, so is there an action I should take here? Or is this conversation resolved?

Reply:
Oh, sorry. Correct, nothing to do here.

"stet" = "I requested a change, but actually, never mind, let the original stand"

Reply:
TIL 😄